sdks/python/apache_beam/metrics/cells.py

```diff
@@ -348,7 +350,17 @@ def combine(self, other):
   def singleton(value, timestamp=None):
     return GaugeData(value, timestamp=timestamp)
 
-  #TODO(pabloem) - Add to_runner_api, and from_runner_api
+  def to_runner_api(self):
+    seconds = int(self.timestamp)
+    nanos = int((self.timestamp - seconds) * 10**9)
+    gauge_timestamp = timestamp_pb2.Timestamp(seconds=seconds, nanos=nanos)
+    return beam_fn_api_pb2.Metrics.User.GaugeData(
+        value=self.value, timestamp=gauge_timestamp)
+
+  @staticmethod
+  def from_runner_api(proto):
+    gauge_timestamp = proto.timestamp.seconds + proto.timestamp.nanos / 10**9
```

Comment:
It looks like this could be doing a truncating division so that the resulting `gauge_timestamp` is an integer and drops the nanosecond values.  Do we want to preserve nanoseconds by potentially doing something like `float(proto.timestamp.nanos)` before doing the division?

Reply:
You are correct Charles. I'll get that fixed now.